notebooks/tutorial_ivf_pq.ipynb

```diff
@@ -0,0 +1,1226 @@
+{
```

Comment:
<blockquote>For the given data set (SIFT-128) and the selected parameters, one can see that 5000 clusters is clearly too much </blockquote><p>I think we should avoid assuming that the user reading this tutorial is familiar w/ reading qps@recall charts. Maybe instead of "one can see that 5000 cluster is clearly too much" we can say something "In this chart you can different queries per second for different levels of recall, calculated by training the index with different n_lists and querying with different numbers of probes. The charts demonstrate that 5000 lists is probably too many, since we see the qps for that curve doesn't reach as high as the others. We can also see that 100 lists is probably not enough, since we see its recall doesn't reach as high as the other curves". </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/raft/pull/1544/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
By this point in the notebook we've shown several QPS/Recall charts and gone rather deep into discussion of the performance implications of the search parameters. I think, it should be safe to assume that the reader, who is interested in such details, should already be familiar with the concept :) But I got the point, I'll try to put it a bit more verbose.

Reply:
<p>It's definitely better. Thanks!</p>